usaspending_api/awards/v2/views/award_spending.py

```diff
@@ -74,16 +74,27 @@ def get_queryset(self):
         top_tier_agency_id = Agency.objects.filter(id=awarding_agency_id).first().toptier_agency_id
         queryset = Transaction.objects.all()
 
-        # Filter based on fiscal_year, awarding_agency_id, and category
```

Comment:
Can't put this at the lines above but for line 60/61, `fiscal_year = json_request.get('fiscal_year', None)`, `.get` defaults to `None` if it's not found so it doesn't need to be explicitly set as None for the 2nd parameter.